packages/modal/src/Modal.story.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { text, select, boolean } from '@storybook/addon-knobs';
+import Modal, { ModalSize, useModalState } from './index';
+
+function Example() {
+  const { active, setActive } = useModalState();
+
+  return (
+    <>
+      <button onClick={() => setActive(active => !active)}>Open Modal</button>
+      <Modal
+        active={active}
+        setActive={setActive}
+        usePortal={boolean('usePortal', true)}
```

Comment:
Hmm... I suppose that we can expose this but I feel a lot of people may shoot themselves in the foot by setting false here.

@hswolff Is there any reason we'd really want to expose this for modal?

Reply:
I think we should just remove the implementation around `usePortal`.